app/state_machines/efile_submission_state_machine.rb

```diff
@@ -120,21 +120,26 @@ class EfileSubmissionStateMachine
     tax_return.transition_to(:file_accepted)
 
     benefits = Efile::BenefitsEligibility.new(tax_return: tax_return, dependents: submission.qualifying_dependents)
-    ctc_amount = benefits.ctc_amount / 2
-    refund_amount = client.intake.claim_owed_stimulus_money_no? ? 0 : (benefits.eip1_amount + benefits.eip2_amount)
-    eip3_amount = benefits.eip3_amount
+    eip1_and_eip2_amount = [benefits.eip1_amount, benefits.eip2_amount].compact.sum
+    total_refund_amount = [benefits.outstanding_ctc_amount, benefits.outstanding_recovery_rebate_credit].compact.sum
+    eip3_amount_received = tax_return.intake.eip3_amount_received || 0
 
-    # TODO: Update analytics table to report relevant information for tax year
     submission.tax_return.create_accepted_tax_return_analytics!(
-      advance_ctc_amount_cents: ctc_amount * 100,
-      refund_amount_cents: refund_amount * 100,
-      eip3_amount_cents: eip3_amount * 100
+      tax_return_year: TaxReturn.current_tax_year,
+      eip1_and_eip2_amount_cents: eip1_and_eip2_amount * 100,
+      advance_ctc_amount_cents: benefits.advance_ctc_amount_received * 100,
+      outstanding_ctc_amount_cents: benefits.outstanding_ctc_amount * 100,
+      ctc_amount_cents: benefits.ctc_amount * 100,
+      eip3_amount_received_cents: eip3_amount_received * 100,
+      eip3_amount_cents: benefits.eip3_amount * 100,
+      outstanding_eip3_amount_cents: benefits.outstanding_eip3 * 100,
+      total_refund_amount_cents: total_refund_amount * 100,
     )
 
     send_mixpanel_event(submission, "ctc_efile_return_accepted", data: {
```

Comment:
this still seems right to me, but might be worth checking with product if you havent already to see whether there are any other values that they want to receive in mixpanel, or confirm how/if this data is being used.